examples/AerosolActivation/aerosol_activation.jl

```diff
@@ -0,0 +1,276 @@
+μ = zeros(length(G_t));
+
+# And add noise to the generated truth sample.
+for i in 1:n_samples
+    y_t[:, i] = G_t .+ rand(Distributions.MvNormal(μ, Γy))
+end
+
+truth_array = EKP.Observations.Obs(y_t, Γy, observation_data_names)
+
+# One could try for the truth to be a mean of the generated array.
+# Or do the calibration for all individual truth samples and then
+# compute the mean of calibrated parameters.
+# For now we are just taking one truth array member.
+truth_sample = truth_array.samples[1];
+
+# We use 50 ensemble members and do 10 iterations.
+N_ens = 50
+N_iter = 10
+
+initial_par = EKP.EnsembleKalmanProcessModule.construct_initial_ensemble(
+    priors,
+    N_ens;
+    rng_seed
+)
+ekiobj = EKP.EnsembleKalmanProcessModule.EnsembleKalmanProcess(
+    initial_par,
+    truth_sample,
+    truth_array.obs_noise_cov, EKP.EnsembleKalmanProcessModule.Inversion(),
+    Δt=1
+)
+
+# Finally, we can run the Ensemble Kalman Process calibration.
+ϕ_n_values = []
+for n in 1:N_iter
+    θ_n = EKP.EnsembleKalmanProcessModule.get_u_final(ekiobj)
+
+    ϕ_n = mapslices(
+        x -> EKP.ParameterDistributionStorage.transform_unconstrained_to_constrained(priors, x),
+        θ_n;
+        dims=1
+    )
+
+    G_n = [run_activation_model(ϕ_n[:, i]...) for i in 1:N_ens]
+
+    G_ens = hcat(G_n...)
+    EKP.EnsembleKalmanProcessModule.update_ensemble!(ekiobj, G_ens)
+
+    global ϕ_n_values = vcat(ϕ_n_values, [ϕ_n])
+end
+
+# We define some simple functions for plotting the data.
+function plot_ensemble_scatter(id)
+
+    ensemble_member = range(1, N_ens, length=N_ens)
+
+    ylabel = "TODO"
+    title = "TODO"
+    if id == 1
+        ylabel = "Molar mass [kg/mol]"
+        filename = "molar_mass_scatter.svg"
+    end
+    if id == 2
+        ylabel = "Osmotic coefficient [-]"
+        filename = "osmotic_coeff_scatter.svg"
+    end
+
+    plot(ensemble_member, ϕ_n_values[1][id,1:N_ens],
+          seriestype=:scatter, xlabel="Ensemble Number", ylabel=ylabel, leg=false
+         )
+    for it in range(2, N_iter, length=N_iter-1)
+        it = Int(it)
+        plot!(ensemble_member.+((it-1)*50), ϕ_n_values[it][id, 1:N_ens], seriestype=:scatter, leg=false)
+    end
+
+    current()
+    savefig(filename)
+end
+
+function plot_ensemble_means(id)
+
+    number_of_iters = range(1, N_iter, length = N_iter)
+    means = zeros(N_iter)
+
+    for it in range(1, N_iter, length = N_iter)
+        it = Int(it)
+        means[it] = mean(ϕ_n_values[it][id, 1:N_ens])
+    end
+
+    ylabel = "TODO"
+    title = "TODO"
+    if id == 1
+        ylabel = "Molar mass [kg/mol]"
+        filename = "molar_mas_average.svg"
+    end
+    if id == 2
+        ylabel = "Osmotic coefficient [-]"
+        filename = "osmotic_coeff_average.svg"
+    end
+
+    plot(number_of_iters, means, xlabel = "ensemble number", ylabel = ylabel, label = "enseble mean")
+    hline!([default_params[id]], label="true value")
```

Comment:
```suggestion
    hline!([default_params[id]], label = "true value")
```

so that Julia formatter test does not fail.